### PR TITLE
Updates readme

### PR DESCRIPTION
**Pull request by jcooklin**
Update includes
* Added the python module and command line entry point as the preferred method for installing/loading the plugin  
* The package based installation description was updated 
* Example was reworded
* Animated GIF was updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,41 +18,66 @@ It's used in the [Snap framework](http://github.com/intelsdi-x/snap).
...
+[releases](https://github.com/intelsdi-x/snap-plugin-collector-pysmart/releases)
+page.
 
-* Ensure that S.M.A.R.T. is enabled on the device
+Known issues:
+* Longer plugin load times 
+* A package is only available for Linux X86_64
+
+Since the current default timeout may be exceeded start `snapteld` with the flag
+`--plugin-load-timeout 30` (e.g. `snapteld -t 0 -1 --plugin-load-timeout 30`).
```
Comment (rashmigottipati) [inline README.md]: Minor typo:  Missing "l" option (log level) in `snapteld -t 0 -1 --plugin-load-timeout 30`.

**Comment by rashmigottipati** [comment]
LGTM 👍

File: README.md [hunk 2]
```diff
@@ -220,8 +269,4 @@ There's more than one way to give back, from examples to blogs to code updates.
 ## License
 [Snap](http://github.com/intelsdi-x/snap), along with this plugin, is an Open Source software released under the Apache 2.0 [License](LICENSE).
 
-## Acknowledgements
-* Authors: [Samantha Alt](https://github.com/saalt),
-           [Joel Cooklin](https://github.com/jcooklin)
```
Comment (IzabellaRaulin) [inline README.md]: Why this section is removed? FYI, it still available in contents in the beginning of README (see [here](https://github.com/intelsdi-x/snap-plugin-collector-pysmart/pull/13/files#diff-04c6e90faac2675aa89e2176d2eec7d8R17))

**Comment by kjlyon** [comment]
Some of your links don't work, License, Documentation, Collected Metrics. Please check.

**Comment by katarzyna-z** [comment]
@kjlyon What I noticed is that links to some sections don't work on this branch and they weren't edited in this PR but these links work on current master so it seems that the issue with links is not connected with this PR.

**Comment by kjlyon** [comment]
Good catch @katarzyna-z. This LGTM.

**Pull request merged**